docs/src/content/docs/guides/connectionoptions.mdx

```diff
@@ -0,0 +1,188 @@
+1. An active Twilio account
+1. A phone number purchased from Twilio
+:::
+
+Creating an Ultravox call that works with Twilio is just like creating a WebRTC call, but there are two parameters to the [Create Call](./api/calls/#create-call) command worth special attention:
+
+<table class="w-full">
+    <tr class="w-full">
+        <th class="w-1/12"></th>
+        <th class="w-1/12"></th>
+        <th class="w-10/12"></th>
+    </tr>
+    <tr>
+        <td class="font-mono">medium</td>
+        <td>object</td>
+        <td>Tells Ultravox which protocol to use. <br />For Twilio, must be set to `{"twilio": {}}` and sets the call to use Twilio [Media Streams](https://www.twilio.com/docs/voice/media-streams). Defaults to `{"webRtc": {}}` which sets the protocol to WebRTC.</td>
+    </tr>
+    <tr>
+        <td class="font-mono">initiator</td>
+        <td>string</td>
+        <td>Tells Ultravox who started the call. For outgoing calls, typically set to `"INITIATOR_AGENT"`. Default is `"INITIATOR_USER"`.</td>
+    </tr>
+</table>
+
+Adding these to the request body when creating the call would look like this:
+
+```javascript
+{
+  "systemPrompt": "You are a helpful assistant...",
+  ...
+  "medium": {
+    "twilio": {}
+  },
+  "initiator": "INITIATOR_AGENT"
+}
+```
+
+Ultravox will return a `joinUrl` that can then be used with Twilio for outgoing or incoming calls.
+
+### Outgoing Calls
+
+It only takes two steps to make an outgoing call to regular phone numbers through Twilio:
+<Steps>
+1. **Create an Ultravox Call** → Create a new call (see [above](#creating-a-phone-call-with-twilio)), and get a `joinUrl`.
+
+1. **Initiate Twilio Phone call** → Use the `joinUrl` with a Twilio [`<Stream>`](https://www.twilio.com/docs/voice/twiml/stream).
+
+    ```javascript
+    // Example using the twilio node library
+    const call = await client.calls.create({
+        twiml: `<Response>
+                    <Connect>
+                        <Stream url="${joinUrl}"/>
+                    </Connect>
+                </Response>`,
+        to: phoneNumber, // the number you are calling
+        from: twilioPhoneNumber // your twilio number
+    });
+    ```
+</Steps>
+
+See the [twilio-outgoing-call](https://github.com/fixie-ai/ultradox/tree/main/examples/twilio-outgoing-call) example for more.
+
+This example shows one of the many options Twilio provides for making outgoing calls. Consult the [Twilio docs](https://www.twilio.com/docs) for more details.
+
+### Incoming Calls
+Incoming calls require essentially the same two steps as outgoing calls:
+
+<Steps>
+1. **Create an Ultravox Call** → Create a new call (see [above](#creating-a-phone-call-with-twilio)), and get a `joinUrl`. *Note: for incoming calls you will want to keep `initiator` set to the default ("user").*
+
+1. **Receive Inbound Twilio Phone call** → Use the `joinUrl` with a Twilio [`<Stream>`](https://www.twilio.com/docs/voice/twiml/stream).
+
+    ```xml
+    <!-- Example using a TwiML Bin -->
```

Comment:
This won't work with TwiML Bin since you need a new joinUrl for each call.  Let's call this an example TwiML response instead.

Reply:
done